src/lib/firestore.ts

```diff
@@ -139,4 +139,17 @@ export const updateCalendarAvailability = (
     });
 };
 
+export const updateEventAvailability = (
+    data: EventDataAvailability,
+    eventId: string
+): Promise<void> => {
+    return new Promise((resolve, reject) => {
```

Comment:
Rather than creating a new Promise I think it would be best to just return the updateDoc since that itself returns a promise.

```
const eventDocRef = getDocRef(`/events/${eventId}`);
return updateDoc(eventDocRef, 'availability', data);
```

Reply:
Awesome, I'm still fuzzy on how promises work, so thanks for the input.

Reply:
Thanks for the suggestion! I just made the changes and pushed